src/overrides/_spacing.scss

```diff
@@ -23,40 +23,39 @@ $_spacing-directions: (
 /// indicates that. We might want to add non-responsive ones later.
```

Comment:
The bit about "the 'r' before the scale point indicates that" just needs to come out.

Reply:
Good spot thanks Hanna